hadeoh/EPICMail
# 164470356 Implement user signup

#### What does this PR do?
Creates a user account

#### Description of the task to be completed
-Create a user model
-Create a user controller
-Create a user services
-Create a userdummy data
_Create a user router 
-Have the POST api/v1/auth/signup endpoint working

#### How can this be manually tested?
Using Postman, pick the HTTP action(POST) in the dropdown menu
-Fill in the url version of the endpoint
-Send the url to the server for testing out the endpoints
-The response should bring a status code and token

#### What are the relevant pivotal tracker stories?
164470356 (https://www.pivotaltracker.com/story/show/164470356)
